### ISSUE TITLE
422 can't create post

### ISSUE THREAD

**Issue by jaumard** [HubPress/hubpress.io#312]
I'm trying to create my first post a can't manage to do it... 

```
Failed to load resource: the server responded with a status of 422 (Unprocessable Entity)
app.js:81184 PostsServices - publishPosts error Object
```

**Comment by ElvisMagagula** [comment]
Hey Guys when I try to save I get `Your post was not saved. See your browser's developer console for the cause of the error.`

**Comment by SattaRavi** [comment]
have had few instances where this was acting up from a computer inside a corp network

**Comment by Mogztter** [comment]
Maybe related to #342 ? Are you using `hp-tags` attribute ?

**Comment by jaredmorgs** [comment]
Closing due to lack of response from OP.

Issues with the tag list causing publishing failure because of a trailing comma was resolved in the 0.6.x release.
